### PR TITLE
keymap Ibm m122

### PR DESCRIPTION
**Pull request by dsanchezseco**
added personal keymap to ibm m122 terminal converter

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/converter/ibm_terminal/keymaps/dsanchezseco/config.h [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#pragma once
```
Comment (drashna) [inline keyboards/converter/ibm_terminal/keymaps/dsanchezseco/config.h:1]: This file can be safely removed.

File: keyboards/converter/ibm_terminal/keymaps/dsanchezseco/rules.mk [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Build Options
```
Comment (drashna) [inline keyboards/converter/ibm_terminal/keymaps/dsanchezseco/rules.mk:1]: You only need the options that are being changed, here, really.

File: keyboards/converter/ibm_terminal/keymaps/dsanchezseco/rules.mk [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+UNICODEMAP_ENABLE = no
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no        # Enable WS2812 RGB underlight.
+PS2_USE_USART = yes
+API_SYSEX_ENABLE = no
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
```
Comment (drashna) [inline keyboards/converter/ibm_terminal/keymaps/dsanchezseco/rules.mk:22]:
```suggestion
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
PS2_USE_USART = yes
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/converter/ibm_terminal/keymaps/dsanchezseco/config.h (hunk 1 above)
Comment (dsanchezseco) [reply keyboards/converter/ibm_terminal/keymaps/dsanchezseco/config.h]: deleted

File: keyboards/converter/ibm_terminal/keymaps/dsanchezseco/rules.mk (hunk 2 above)
Comment (dsanchezseco) [reply keyboards/converter/ibm_terminal/keymaps/dsanchezseco/rules.mk:1]: fixed

File: keyboards/converter/ibm_terminal/keymaps/dsanchezseco/rules.mk (hunk 3 above)
Comment (dsanchezseco) [reply keyboards/converter/ibm_terminal/keymaps/dsanchezseco/rules.mk]: fixed

**Review by drashna — APPROVED** [review]

File: keyboards/converter/ibm_terminal/keymaps/dsanchezseco/rules.mk [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+MOUSEKEY_ENABLE = no        # Mouse keys(+4700)
```
Comment (fauxpark) [inline keyboards/converter/ibm_terminal/keymaps/dsanchezseco/rules.mk:5]:
```suggestion
MOUSEKEY_ENABLE = no
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/converter/ibm_terminal/keymaps/dsanchezseco/keymap.c [hunk 5]
```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2016 Priyadi Iman Nurcahyo <priyadi@priyadi.net>
```
Comment (noroadsleft) [inline keyboards/converter/ibm_terminal/keymaps/dsanchezseco/keymap.c:2]: Should be your own name / email / year here IMO.

File: keyboards/converter/ibm_terminal/keymaps/dsanchezseco/keymap.c [hunk 6]
```diff
@@ -0,0 +1,44 @@
+#include QMK_KEYBOARD_H
```
Comment (fauxpark) [inline keyboards/converter/ibm_terminal/keymaps/dsanchezseco/keymap.c:1]: Could you add a license header here?

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Comment by drashna** [comment]
For future reference, we recommend against committing to your `master` branch as you've done here, because pull requests from modified `master` branches can make it more difficult to keep your QMK fork updated. It is highly recommended for QMK development – regardless of what is being done or where – to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your fork updated here:

[**Best Practices: Your Fork's Master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)

[Fixing Your Branch](https://docs.qmk.fm/#/newbs_git_resynchronize_a_branch) will walk you through fixing up your `master` branch moving forward. If you need any help with this just ask.

Thanks for contributing!

**Comment by dsanchezseco** [comment]
Did I?? Sorry, did it in a hurry and didn't notice, my bad.

-------- Original Message --------

**Comment by dsanchezseco** [comment]
> For future reference, we recommend against committing to your `master` branch as you've done here, because pull requests from modified `master` branches can make it more difficult to keep your QMK fork updated. It is highly recommended for QMK development – regardless of what is being done or where – to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.
> 
> There are instructions on how to keep your fork updated here:
> 
> [**Best Practices: Your Fork's Master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)
> 
> [Fixing Your Branch](https://docs.qmk.fm/#/newbs_git_resynchronize_a_branch) will walk you through fixing up your `master` branch moving forward. If you need any help with this just ask.
> 
> Thanks for contributing!

Just checked and i didn't committed to master, I pulled master and merged into the branch i was using, but never committed to it

**Pull request merged**